### PR TITLE
readme v1

### PR DESCRIPTION
**Pull request by sguaschsbt**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+The unittests and integration tests are in an optional part of the code, and are written in C++.  These tests rely on 
+[```benchmark```](https://github.com/google/benchmark), and ```gtest``` libraries, and therefore require a C++17 compiler.
+
+Currently, the project has been tested with the gcc,g++ >= 11.3.0 compiler under Linux (x86_64). In the future, we plan to
+extend the compatibility to other compilers, platforms and operating systems. 
+
+
+## Installation
+
+To build the library with the default options, create a ```build``` folder in the top level directory and run ```make``` from inside it. This assumes that the standard tool cmake is already installed on the system, and an up-to-date c++ compiler (i.e. g++ >=11.3.0) as well.
```
Comment (ngama75) [inline README.md]: The library uses a classical `cmake` build mechanism: use cmake to create a ```build``` folder in the top level directory and run ```make``` from inside it.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+| ENABLE_TESTING | Compiles unit tests and integration tests |
+
+
+
+## Usage
+
+### Layouts for complex FFT
+
+The complex FFT handles polynomials with complex coefficients `C[X] mod X^M-i`, each one represented by two consecutive doubles `(real,imag)`. Note that a real polynomial $$\sum_{j=0}^{N-1} p_j\cdot X^j \mathtt{ mod } X^N+1$$ corresponds to the complex polynomial of half degree `M=N/2`: 
+$$\sum_{j=0}^{M-1} (p_{j} + i.p_{j+M}) \cdot X^j \mathtt{ mod } X^M-i$$
```
Comment (ngama75) [inline README.md:59]: add:
In both cases, the FFT of the polynomial is the vector of its (complex) evaluations on the $M$ roots of $X^M-i$.

**Review by ngama75 — APPROVED** [review]
Sounds good!!

File: README.md (hunk 2 above)
Comment (sguaschsbt) [reply README.md:59]: added

**Pull request merged**